packages/integration-tests/src/test-project-setup/conversation_handler_project.ts

```diff
@@ -158,7 +210,130 @@ class ConversationHandlerTestProject extends TestProjectBase {
         },
       ],
       request: {
-        headers: { authorization: authenticatedUserCredentials.accessToken },
+        headers: { authorization: accessToken },
+      },
+      responseMutationInputTypeName:
+        'CreateConversationMessageAssistantResponseInput',
+      responseMutationName: 'createConversationMessageAssistantResponse',
+      modelConfiguration: {
+        modelId: 'anthropic.claude-3-haiku-20240307-v1:0',
+        systemPrompt: 'You are helpful bot.',
+      },
+    };
+    await this.executeConversationTurn(
+      event,
+      defaultConversationHandlerFunction,
+      apolloClient
+    );
```

Comment:
we don't make assertions on the response here, is this because we are just passing the request to bedrock (which might produce inconsistent result?)

Reply:
Correct.
There is a basic assertion inside `this.executeConversationTurn` which checks that some response was emitted. But we do not inspect the answer as it might not be very stable. This could perhaps be changed if we use more concrete question. I'll give it a shot.

Reply:
I found something that should be stable.